src/CodeGenerator/Generator/GeneratorHelper.php

```diff
@@ -27,11 +27,14 @@ public static function toPhpType(string $parameterType): string
             case 'boolean':
                 return 'bool';
             case 'integer':
+            case 'long':
```

Comment:
Are you sure you can do this? What if a number larger than INT_MAX is passed?

Reply:
You're right. The official SDK use "int" for documentation, but cast to string.

I reverted that part